modules/representation_management/app/models/representation_management/form_2122_base.rb

```diff
@@ -100,12 +100,29 @@ class Form2122Base
       validates :claimant_phone, length: { is: 10 }, format: { with: TEN_DIGIT_NUMBER }
     end
 
-    validates :representative_id, presence: true
-
     def representative
       @representative ||= find_representative
     end
 
+    def representative_individual_type
+      type = if representative.is_a?(AccreditedIndividual)
+               representative.individual_type
+             else
+               representative.user_types.first
+             end
+      # We're converting 'claims_agent' and 'claim_agents' to 'agent'
+      # here because the PDF checkbox responds to 'agent'.
+      type&.include?('agent') ? 'agent' : type
```

Comment:
what do you think about an exact match:
```suggestion
      ['claims_agent', 'claim_agents'].include?(type) ? 'agent' : type
```

Reply:
Sounds good!